modular_skyrat/modules/window_airbags/code/window_airbag.dm

```diff
@@ -0,0 +1,101 @@
+/obj/structure/window/reinforced/fulltile/Initialize(mapload, direct)
```

Comment:
So even if an engi creates a window, it'll make the airbag element?

Reply:
Yep, seems okay?